src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/OpportunitiesOrchestrator.cs

```diff
@@ -374,10 +371,18 @@ public async Task<ApplicationDetailsViewModel> GetApplicationViewModel(Applicati
 
         public async Task<ApplicationRequest> PostApplicationViewModel(ApplicationDetailsPostRequest request)
         {
-            var applicationDetails = await _opportunitiesService.GetApplicationDetails(request.PledgeId);
+            var applicationDetails = await _opportunitiesService.GetApplicationDetails(request.PledgeId, request.SelectedStandardId);
+
+            request.Amount = (await GetFundingEstimate(new GetFundingEstimateRequest
```

Comment:
I think it'd be worth holding this in a variable and passing it into the `UpdateCacheItem` method as a seperate parameter.